docs/can-guides/contribute/site-style-guide.md

```diff
@@ -0,0 +1,87 @@
+@page guides/contributing/site-style-guide Site Style Guide
+@parent guides/contribute 7
+
+@description
+This is the CanJS Site Style Guide
+
+@body
+
+<h2>Headings</h2>
+<h2>H2</h2>
+<p>Paragraph</p>
+<h2>H2 with <code>code</code></h2>
+<h3>H3</h3>
+<p>Paragraph</p>
+<h4>H4</h4>
+<p>Paragraph</p>
+<h5>H5</h5>
+<p>Paragraph</p>
+<h6>H6</h6>
+<p>Paragraph</p>
+<h2>Block elements</h2>
+<p>A paragraph. Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur.</p>
+<p>Another paragraph. Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.</p>
+<ul>
+<li>List element</li>
+<li>Another item</li>
+<li>Another item</li>
+</ul>
+<p>Another paragraph. Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.</p>
+<ul>
+<li><p>List element</p>
+<ul>
+<li>nested</li>
+<li>next
+<ul>
+<li>deep
+<pre><code>foo
+</code></pre></li>
+</ul></li>
+</ul></li>
+<li><p>Another item</p></li>
+<li><p>Another item</p>
+<p>multiple paragraphs</p></li>
+</ul>
+<p>Another paragraph. Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.</p>
+<ol>
+<li>Order lists</li>
+<li>Number two</li>
+<li>Number three</li>
+</ol>
+<blockquote>
+<p>Blockquote. Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.</p>
+</blockquote>
+<p><details>
+<summary>This is a summary element that shows a paragraph</summary></p>
+<p>Another paragraph. Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.</p>
+<p></details></p>
+<p><details>
+<summary>This is a summary element that shows a pre</summary></p>
+<pre><code class="language-js">foo();
+</code></pre>
+<p></details></p>
+<p>HTML code example:</p>
+<pre><code class="language-html">&lt;hello-world&gt;&lt;/hello-world&gt;
+</code></pre>
+<p>JS code example:</p>
+<pre><code class="language-js">console.log(&quot;Hello There&quot;)
+</code></pre>
+<p>CodePen example:</p>
+<pre><code class="language-js">console.log(&quot;Hello There&quot;)
+</code></pre>
+<div class='codepen'></div>
```

Comment:
For the CodePen integration, I think it’d be better to have a real example in Markdown because this structure could change (will change?). Otherwise, we’d have to update the style guide _and_ make changes in the bit-docs package.

Reply:
@chasenlehara Can you provide a CodePen example in Markdown?

Reply:
We use the `@codepen` tag to make an example able to be opened in CodePen.

An example from: https://github.com/canjs/canjs/blob/91bce285dd8d471541f7d92dedada200ef94b778/docs/can-guides/topics/data/managing-sessions/managing-sessions.md#configuring-the-behavior

```md
@sourceref ./cookie-session-basic.html
@highlight 22-27,only
@codepen
```